### DISCUSSION TITLE
Equipo 11 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#17] [Proyecto Final]
Hola @PilyRmz 
Aquí está el hilo en el foro para tu equipo. Podrías conversar aquí sobre tus avances con el proyecto y avisar al Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengas al respecto.
Con la emoción de seguir el desarrollo de tu proyecto,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @PilyRmz, buenas tardes,

Gracias por entregar la etapa 0 de tu proyecto final. Estoy revisándola, y te escribo por este medio con retroalimentación. En primer lugar, me parece un foco muy impactante. Gracias por dar el contexto/el razonamiento por la cual elegiste enfocarte en este asunto para tu proyecto final, suena super relevante para tu comunidad.

Escribo con varias sugerencias. Las preguntas dentro del protocolo son bien construidas. Sugiero empezar con algo para romper el hielo (una pregunta, una historia, lo que sea). Sería importante avisar a la entrevistada/o sobre la duración de la entrevista _antes_ de empezar la entrevista, para que esté preparada. Además, una posible sugerencia de mi parte sería preguntar sobre los problemas que encuentren el entrevistado/a respecto al acceso de agua, para que tengas algunas historias (en adición que tu experiencia personal, por supuesto) que dirigen tu ideación la semana que viene. También, ¿qué opinas de agregar una pregunta que identifica los papeles que tiene, por ej, el gobierno local/municipal, las ONGs, cada familia, cada edificio donde vive la gente, etc? Quizás una pregunta así te apoye a idear sobre el problema la semana que viene.

¿Has elegido a una entrevistado/a? Recomiendo identificar a alguien, agendar y completar la entrevista tan pronto como sea posible, para tener suficiente tiempo esta semana para sintetizar tus apuntes durante la entrevista y entregar la etapa 1 del proyecto.

¿Tienes preguntas para mí? Cualquier cosa, aquí ando.

¡Suerte!
Saludos,
Kasturi

**Comment by PilyRmz** [comment]
Hola, muchas gracias Kasturi, te agradezco la retroalimentación me sirvio mucho. Ya hice una modificicación a mis preguntas para la entrevista. :)

**Reply by KasturiSShah** [reply]
Hola @PilyRmz 
Justo vi las actualizaciones. Me parece bien la pregunta para romper el hielo, y la última pregunta que añadiste sobre acciones que individuos podrían tomar respecto al agua.
Mucha suerte para la entrevista, y feliz día de la mujer,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @PilyRmz 

Justo miré tu etapa 1 del proyecto final (aquí https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-11/etapa-1/README.md) y no veo la entrega ahí. Perdón si me equivoqué. ¿Cómo andas con tu entrevista y la creación de la mapa de síntesis? ¿Necesitas una prórroga? Te podría dar una prórroga hasta este domingo.

Quedo al pendiente,
Saludos,
Kasturi

**Comment by PilyRmz** [comment]
Hola Kasturi,
Disculpa la demora, y sí perdón voy atrasada en las tareas. Se me ha
complicado un proyecto q me pidieron en el trabajo.
Pensaba el domingo ya terminar y subir lo que me falta de las tareas, si es
posible?

Gracias por tu tiempo y ayuda :)

**Reply by KasturiSShah** [reply]
Hola @PilyRmz 
Gracias por la actualización, y no te preocupes. Claro que sí, tienes prórroga hasta este domingo. Mucha suerte con tu proyecto. Me avisas si necesitas más tiempo.
Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @PilyRmz 

Espero estés muy bien y que pasaste una feliz Semana Santa. Gracias por entregar la etapa 1 y 2 del proyecto final. Las ideas capturas de pantalla de tu proceso del 8S y SCAMPER me parecen creativas y diferentes. Ahora es momento de pensar en su prototipado baja fidelidad. Te escribo con algunas sugerencias:

1. Para la sintetización de apuntes tomados durante las entrevistas, te animo a agrupar las respuestas de ambas entrevistadas por tema en vez de por pregunta. O, crear una mapa de empatía para cada entrevistada. Sin embargo, dado las alturas del curso en que estamos, sugiero que regreses a esta sugerencia cuando el curso termine (solo para practicar cómo crear estas mapas) y que - por el momento - sigas adelante con la etapa 3 y 4.
2.  En general, cada una de tus ideas en la captura de pantalla llamada "Ideación SCAMPER" puede ser un proyecto. Para el prototipado, recomiendo crear un prototipado sobre 1 (o cómo máximo 2) de ellas para que puedas avanzar con el resto de las etapas. ¿En qué idea quieres enfocarte? (¿Sería la captación de agua pluvial? Pregunto porque veo diagramas en Canva al respecto)
3. ¿Cúal sería tu prototipo low fidelity? Debe ser un dibujo sencillo de tu producto final. Por ejemplo, puede ser una diagrama de baños secos o un "plan de estudios" para el taller de concientización. Solo son sugerencias, no necesitas usarlas 😄 
4. En Canva veo algunas diagramas de captación de lluvia pluvial. ¿Creaste estas imágenes o las buscaste desde fuentes en linea? Siempre es importante dar crédito a los fuentes, y te animo a insertar la liga a fuentes que usaste en tus diapositivas en Canva
5. Una pregunta: ¿hasta qué fecha quieres una prórroga para la etapa 3? Y cómo andas con la etapa 3? Me avisas cualquier cosa.
6. En Canva veo las grabaciones de las entrevistas; no es necesario compartirlas 😄 Si son para tu futuro uso/referencia y por eso están ahí, entiendo. Solo quisiera aclarar que no pedimos que grabes las entrevistas.

Cualquier cosa, aquí ando. Suerte con la etapa 3 y hasta pronto.

Saludos,
Kasturi

**Comment by PilyRmz** [comment]
Hola Kasturi ☺️
Disculpa la demora, para contestarte.
No lo había hecho porque según yo, estaba planeando terminar ya ayer o
máximo hoy la etapa 3. Siento que no es tanto lo que se tiene que hacer y
estoy conciente.

Se me ha complicado las semanas por trabajo y luego me caí tuve un esguince
en el pie, aún así no es justificación pude organizarme mejor.

Mi plan era acabar ya la etapa 3, con un prototipo de sistema de
capacitación de agua de lluvia para las casas.
Por eso lo había puesto en el vídeo de la etapa 2, pero decidí quitar esas
imágenes, acotar la etapa 2. Y en el prototipo de la etapa 3, hacer mi
propio diseño/dibujo del sistema de capacitación de agua, y explicarlo.

El sábado me conecto para la asesoría y una disculpa, no sé si haya
problema que a más tardar el día domingo, ya entregue todo para estar al
corriente, se q nos han dado mucho tiempo, en serio no pensé q se me
complicaría mis tiempos.

Pero prometo el domingo más tardar, ya estar al corriente con todo.

Muchas gracias, también por el apoyo.
Lindo fin de semana 😊

**Reply by KasturiSShah** [reply]
Hola @PilyRmz 

Espero te encuentres mucho mejor. No hay nada que disculpar, entiendo completamente. Veo tus etapas en Github ahora, las estoy revisando y te escribo en unos minutos con retroalimentación. ¡Felicidades por ponerte al corriente tan rápidamente!

Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @PilyRmz 

Gracias por entregar tu etapa 3 y 4. Felicidades otra vez por ponerte al corriente con el trabajo rápidamente. Sobre todo, a mí me gusta mucho los alineamientos y el flujo entre el prototipado baja fidelidad y alta fidelidad.

Algunas recomendaciones / pensamientos:
- ¿Cómo creaste el prototipo alta fidelidad? Una sugerencia de mi parte sería escribir en la diapositiva "Hecho en XXX" para que se quede todo claro. También puede inspirar a otras alumnas durante la clausura del curso. 😉 
- ¿Has platicado con un usuario potencial sobre tu diseño? Es muy importante hacerlo, de esa manera puedes hacer ajustes al prototipo. Una posibilidad sería platicar con la persona que entrevistaste en la etapa 1. O otra opción sería platicar con un ingeniero civil que te pueda dar consejos.
- Una cosa que te recomiendo contemplar: ¿qué pasaría si no hubiera mucha lluvia en la comunidad que tiene captación pluvial? Sugiero que añadas una diapositiva para la clausura pensando en alternativas que cada casa puede usar: por ejemplo, guardando agua cada mes o usando las fuentes de agua que ya existe. De hecho, podrías escribir en la plantilla Sprint para la etapa 5 que una de las tareas para el futuro sería investigar alternativas cuando no se hace mucha lluvia.
- Otra cosa para contemplar: ¿cuales serían las pautas para el uso del agua? Por ejemplo, la cantidad de agua que alguien puede usar diariamente sin vaciar el tanque, los usos para este agua (es decir, es agua potable solamente [beber, cocinar] o también se puede usarlo para duchar? Podría ser una manera de restringir el uso del agua y guardarlo para las temporadas secas).
- ¿Estás pensando en compartir pautas para el mantenimiento del sistema también? Pregunto porque no solo es importante construir sistemas así, sino también importante pensar en sus "vidas" y dar entretenimiento a los usuarios para que puedan mantenerlo. 

Finalmente ...
¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar su presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si les conviene la asignación de su equipo a una de las dos sesiones de clausura del curso, o si quieren cambiar su día asignado al otro día.

Estoy aquí para apoyarte, déjame saber cualquier pregunta sobre el proyecto por este medio.

Saludos, mucho éxito y mucha suerte, feliz fin,
Kasturi

**Comment by PilyRmz** [comment]
Hola Kasturi :)

Gracias por las recomendaciones, ya las tengo en cuenta para mi
presentación del miércoles.
Si me queda bien, ese día miércoles. No pude hacer más pruebas con el
prototipo pero hable con la persona entrevistada 1

Nos vemos el miércoles para la clausura
Abrazos, buen día
